Makefile

```diff
@@ -0,0 +1,16 @@
+TYPESCRIPT_URL = https://github.com/Microsoft/TypeScript.git \
+
+.PHONY: install
+install:
+	npm install -g typescript
+
+.PHONY: build
+build:
+	git clone $(TYPESCRIPT_URL)
```

Comment:
If this file is part of the repo they'd have to had already cloned the repo to get this file.

Reply:
Excuse me. I had not tried that.